lib/common/statusFileCreator.py

```diff
@@ -0,0 +1,113 @@
+                                                            900: "Successfully complete"},
+                           STATUS_TAG_SLACK: {100: "Get command from slack channel",
+                                              200: "Put command into local queue",
+                                              900: "Successfully complete"},
+                           STATUS_TAG_INTERACTIVE: {100: "Get command from interactive mode",
+                                                    200: "Put command into local queue",
+                                                    300: "Execute scheduler realted command",
+                                                    900: "Successfully complete"}
+                           }
+
+    @staticmethod
+    def get_status_folder():
+        """
+        Get status folder path and create folder if not exist
+        @return: status folder path
+        """
+        default_hasal_repo_path = CommonUtil.auto_get_hasal_repo_path()
+        status_folder_path = os.path.join(default_hasal_repo_path, StatusFileCreator.DEFAULT_STATUS_FOLDER_NAME)
+        if not os.path.exists(status_folder_path):
+            os.mkdir(status_folder_path)
+        return status_folder_path
+
+    @staticmethod
+    def create_job_id_folder(input_job_name):
+        """
+        create job id folder
+        @param input_job_name:
+        @return: job id
+        """
+        # check status folder exist
+        status_folder_path = StatusFileCreator.get_status_folder()
+        utc_timestamp = CommonUtil.get_utc_now_timestamp()
+        job_id = "%s-%s" % (input_job_name, utc_timestamp)
+        job_id_folder_path = os.path.join(status_folder_path, job_id)
+
+        if os.path.exists(job_id_folder_path):
+            logger.error("Failed on creating job id folder due to job id duplicate, job_id_folder_path:[%s]" % job_id_folder_path)
+            return None
+        else:
+            os.mkdir(job_id_folder_path)
+            return job_id
+
+    @staticmethod
+    def get_status_code_desc(status_tag, status_code):
+        if status_tag not in StatusFileCreator.STATUS_CODE_MAPPING:
+            logger.error("Cannot find status tag[%s] in current status code mapping [%s]" % (status_tag, StatusFileCreator.STATUS_CODE_MAPPING))
+        else:
+            if status_code not in StatusFileCreator.STATUS_CODE_MAPPING[status_tag]:
+                logger.error("Cannot find status code[%s] in current status tag dict [%s]" % (status_code, StatusFileCreator.STATUS_CODE_MAPPING[status_tag]))
+            else:
+                return StatusFileCreator.STATUS_CODE_MAPPING[status_tag][status_code]
+        return None
+
+    @staticmethod
+    def create_status_file(status_file_folder, status_tag, status_code, status_content=None):
+        """
+        create status file, the status file format will be 123-xyz.json
+        @param status_file_folder:
+        @param status_tag:
+        @param status_code:
```

Comment:
please adding description of `status_code` for telling developer it will relate to `STATUS_CODE_MAPPING`.